src/classes/issues-processor.ts

```diff
@@ -911,6 +922,48 @@ export class IssuesProcessor {
     return this.options.removeStaleWhenUpdated;
   }
 
+  private async _removeLabelsWhenUpdatedFromStale(
+    issue: Issue,
+    labels: Readonly<string>[]
+  ): Promise<void> {
+    if (!labels.length) {
+      return;
+    }
+
+    const issueLogger: IssueLogger = new IssueLogger(issue);
+
+    issueLogger.info(
+      `Removing any labels specified via remove-labels-when-updated-from-stale...`
+    );
+
+    for (const label of labels.values()) {
+      await this._removeLabel(issue, label);
+    }
+  }
+
+  private async _addLabelsWhenUpdatedFromStale(
```

Comment:
Add tests, please?

Reply:
See https://github.com/actions/stale/pull/468#issuecomment-854177606, should have replied to this comment.